src/org/labkey/cds/view/template/FrontPage.jsp

```diff
@@ -62,10 +62,43 @@
     <%--<!-- Client API Dependencies -->--%>
     <script type="text/javascript" src="<%=text(contextPath)%>/clientapi/core/Utils.js"></script>
     <script type="text/javascript" src="<%=text(contextPath)%>/clientapi/core/ActionURL.js"></script>
+    <script type="text/javascript" src="<%=text(contextPath)%>/clientapi/core/Ajax.js"></script>
+    <script type="text/javascript" src="<%=text(frontPagePath)%>/components/jquery/dist/jquery.min.js"></script>
     <script type="text/javascript">
         reloadRegisterPage = function() {
             window.location = LABKEY.ActionURL.buildURL('cds', 'app', LABKEY.container.path, {register: "TRUE"});
-        }
+        };
+
+        LABKEY.Utils.onReady(function ()
+        {
+            $(document).ready(function(){
+
+                // notification close button
+                $('div.dismiss').click(function(){
+                    $('#notification').remove();
```

Comment:
This removes it from the UI, but do we need to call the core-dismissCoreWarnings API at all from the FrontPage.jsp like we do for Header.js below?

Reply:
probably not, since there is no session established at this point